### ISSUE TITLE
Changelog no longer renders in GitHub

### ISSUE THREAD

**Issue by guseggert** [ipfs/go-ipfs#8976]
### Checklist

- [X] This is a bug report, not a question. Ask questions on [discuss.ipfs.io](https://discuss.ipfs.io).
- [X] I have searched on the [issue tracker](https://github.com/ipfs/go-ipfs/issues?q=is%3Aissue) for my bug.
- [X] I am running the latest [go-ipfs version](https://dist.ipfs.io/#go-ipfs) or have an issue updating.

### Installation method

built from source

### Description

https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md

It's too long. I propose we split it into e.g. `CHANGELOG.1.md`, `CHANGELOG.2.md`, etc. (higher number == more recent), and `CHANGELOG.md` which contains the most recent changes.

**Comment by BigLep** [comment]
@ajnavarro @Jorropo : given comments in https://github.com/ipfs/go-ipfs/pull/8997#issuecomment-1173674698, I think we need to take care of this this week for the release process.

**Comment by Jorropo** [comment]
@BigLep I agree, what about just moving the 0.13.0 changelog in it's own file and we just link to it in the main changelog file ?

**Comment by BigLep** [comment]
@Jorropo : isn't it just that 0.13 "overfilled the bathtub" but weren't we already close to overflowing?  

What about doing a changelog per year where everything in 2022 (including 0.13)  would be in CHANGELOG.md.  

It doesn't really matter to me how we solve it, but lets solve it so we don't have more conversation on this year or next.

**Comment by Jorropo** [comment]
@BigLep fair enough, I'll send a PR for per year changelog.
